downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -0,0 +1,37 @@
+ifdef::context[:parent-context: {context}]
+
+[id="using-rhsso-operator-with-automation-hub"]
+
+:context: using-a-rhsso-operator
+
+= Using a Red Hat Single Sign On Operator with {HubName}
```

Comment:
```suggestion
= Using Red Hat Single Sign On Operator with {HubName}
```

Reply:
I think all RH product naming should go into attributes rather than hard-coded. RHSSO might change to "Red Hat Build of Keycloak" or something else.

Reply:
@oraNod @ianf77 
Is there an official name for SSO?
In the [Official product and solution names list](https://docs.google.com/spreadsheets/u/2/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/pubhtml?gid=1259317633&single=true), they mention "Red Hat's single sign-on (SSO) technology"

Reply:
Right now the official name is "Red Hat single sign-on". I am aware that there are discussions to change the official name to "RH Build of Keycloak". I do think all RH product names need to go into an attribute, perhaps `{RHSSO}`.